docs/acceptance.md

```diff
@@ -0,0 +1,42 @@
+<img width="840px" height="auto" src="./acceptance-wires.png" alt="Splitting basics">
+
+### In General
+- [ ] It should generally match Photon styles.
+- [ ] It should have a way for users to provide feedback without leaving the experiment.
+- [ ] It should match final visual specs.
+
+### Context Clicks
+- [ ] It should register a _link_ context menu item for opening sites in the sidebar
+- [ ] It should register a _page_ and _tab_ context menu item for opening sites in the sidebar
+- [ ] Both context menu buttons should include the extension icon
+
+### Sidebar Homepage
+- [ ] It should have a homepage
+- [ ] It should have disimissable onboarding content (text + icon for MVP)
+- [ ] It should show a list of currently open tabs
+- [ ] It should show a list of recently sidebarred tabs
```

Comment:
I think this needs a small `clear` link to empty the list.  Also, call out ignoring private mode urls too?